### PR TITLE
Moved pack to Beta and to community supported

### PR DESCRIPTION
**Pull request by maimorag**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://jira-hq.paloaltonetworks.local/browse/CIAC-4287

## Description
1. Verify and review integration documentation with regards to actual version it was tested on and hence working, e.g 10.1.12.
2. Move pack to Beta and to community supported

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [x] 6.0.0

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/PANOSPolicyOptimizer/pack_metadata.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-    "name": "PAN-OS Policy Optimizer",
+    "name": "PAN-OS Policy Optimizer (beta)",
     "description": "This integration introduces Policy Optimizer and DAG features that are not available through the regular PAN API",
     "support": "xsoar",
```
Comment (Shellyber) [inline Packs/PANOSPolicyOptimizer/pack_metadata.json:4]: Need to update the support for community as well

Comment (Shellyber) [reply Packs/PANOSPolicyOptimizer/pack_metadata.json:4]: Also please fix validation - add new release notes and bump the pack version

Comment (maimorag) [reply Packs/PANOSPolicyOptimizer/pack_metadata.json]: fixed - updated the cuuerntVersion and support in metadata file.

File: Packs/PANOSPolicyOptimizer/Integrations/PANOSPolicyOptimizer/README.md [hunk 2]
```diff
@@ -1,9 +1,9 @@
 Automate your AppID Adoption by using this integration together with your Palo Alto Networks Next-Generation Firewall or Panorama.
 This integration was integrated and tested with version 8 up to 10.1.6 of PAN-OS Policy Optimizer, later versions may not work
```
Comment (Shellyber) [inline Packs/PANOSPolicyOptimizer/Integrations/PANOSPolicyOptimizer/README.md:2]: Are you sure about the versions?

**Review by Shellyber — COMMENTED** [review]
Looks good, just fix the validation :)

File: Packs/PANOSPolicyOptimizer/Integrations/PANOSPolicyOptimizer/README.md (hunk 2 above)
Comment (maimorag) [reply Packs/PANOSPolicyOptimizer/Integrations/PANOSPolicyOptimizer/README.md:2]: I made sure with Gal (we had a meeting last Thursday) it was the previous version it was tested on.
I'm waiting for a new instance (https://jira-hq.paloaltonetworks.local/browse/CRTX-67271 ) to check it on version 10.1.12

**Review by Shellyber — APPROVED** [review]

**Comment by bakatzir** [comment]
DId I miss it, or did we not add a note WHY it is beta? (no formal PAI, it was scraped)
please work with @ShirleyDenkberg to add it to the reamde.

**Pull request merged**